content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx

```diff
@@ -104,13 +110,38 @@ Note:
 
 - A page omitted from search results is still shown in the navigation. To hide it in the navigation, use the [`hidePage`](#hidepage) tag.
 
-<!--
+## <inlinecode>langSwitcher</inlinecode>
+
+Accepts an array of languages to be shown in the page as dropdown options in order to switch the context between the pages
+
+Example:
+
+```
+langSwitcher: ['typescript', 'node']
+```
+
+## <inlinecode>dbSwitcher</inlinecode>
 
-  langSwitcher?: string[]
-  dbSwitcher?: string[]
-  staticLink?: boolean
```

Comment:
Does not exist any more?

Reply:
Ah no, was just already docuemnted before.

Reply:
These 3 options have been documented already